addons/web/i18n/web.pot

```diff
@@ -43,6 +43,17 @@ msgstr ""
 msgid " records"
 msgstr ""
 
+#. module: web
+#: model_terms:ir.ui.view,arch_db:web.styles_company_report
+msgid ""
+"!important;\n"
```

Comment:
@kebeclibre code ends up in translated content, `t-translation="off"` to add probably, introduced at 7c488b7180f620e28bc747c9b85f630d0b2b798a